app/js/components/overview/widgets/newRules/newRules.jade

```diff
@@ -0,0 +1,17 @@
+.grid-block.new-rules(ng-show="rules.length")
+  section.widget-block.newRules
+    h2.title
+      i.fa.fa-institution
+      span(translate) Rules Added in the past 15 days
+      a.cta-link.pull-right(ui-sref='app.rules', translate) View rules
+    .content
+      .spinner(ng-show='loader.loading')
+      p(translate) {{rules.length}} new rules
+      .rule-wrapper(ng-repeat="rule in rules | orderBy: 'rule.created_at' | limitTo: 3")
+        .rule-title
+          a(ui-sref='app.rules({"anchor" : rule.rule_id})') {{rule.description}}
+        .rule-date
+          span.light(translate) Published {{rule.publish_date | timeAgo}}
+    .footer.text-center
+      a(ng-if="rules.length === 1", ui-sref='app.rules({age: 15})', translate) View the new rule
+      a(ng-if="rules.length > 1", ui-sref='app.rules({age: 15})', translate) View all {{rules.length}} new rules
```

Comment:
This looks like it should work to me but when I click on it I get the full rule list on the rules page.

Reply:
Really?? I wonder how I can reproduce.

Reply:
Still running into this. To reproduce:

1) Go to overview for account 540155
2) Select view all 5 new rules on the Rules Added in the Past 15 Days
3) Scroll to the bottom of the rules page and observe that there are 5+ pages of rules instead of just 5 rules.